libs/ebpfnetsh/maps.cpp

```diff
@@ -39,12 +40,16 @@ handle_ebpf_show_maps(
     std::cout << "     ID            Map Type  Size   Size  Entries     ID  Pins  Name\n";
     std::cout << "=======  ==================  ====  =====  =======  =====  ====  ========\n";
 
-    uint32_t map_id = 0;
-    for (;;) {
-        if (bpf_map_get_next_id(map_id, &map_id) < 0) {
-            break;
+    std::vector<ebpf_id_t> map_ids;
+    {
+        uint32_t map_id = 0;
+        while (bpf_map_get_next_id(map_id, &map_id) == 0) {
+            map_ids.push_back(map_id);
         }
+    }
+    std::sort(map_ids.begin(), map_ids.end());
```

Comment:
This sort shouldn't be needed since bpf_map_get_next_id is specified as returning IDs in numerical order.